phpBB/legacy_front/download/file.php

```diff
@@ -299,6 +225,7 @@
 	{
 		wrap_img_in_html(append_sid($phpbb_root_path . 'download/file.' . $phpEx, 'id=' . $attachment['attach_id']), $attachment['real_filename']);
 		file_gc();
+		return;
 	}
 	else
```

Comment:
actually the else isn't useless but the return is (considering the workflow here) :)
But I prefer to explicitly keep the return as file_gc() used to call exit() (which I have removed)